app/views/site/_study_download_data.html.erb

```diff
@@ -1,3 +1,5 @@
+<%= render partial: 'study_download_agreement' if @download_agreement.present? %>
```

Comment:
Do we still want to render the agreement if it's expired?

Reply:
No, great catch.

Reply:
ef48f0b